index.html

```diff
@@ -21,6 +21,7 @@
         <nav class="navbar">
           <a href="#" class="logoo">Tuzy Orizu</a>
           <div class="place hide">
+            <span class="close"></span>
```

Comment:
- [ ] Please make sure to use the proper linters for this milestone (As this is your first time using the JS linters it would be required, but in the future, this can invalidate your project). You can use [these linters](https://github.com/microverseinc/linters-config/tree/master/html-css-js) provided by Microverse. Also, please make sure they pass with no errors.
- [ ] Kindly make sure that if the user clicks on a nav link on the desktop navbar or the mobile menu navigation links, the page is scrolled to the appropriate part (When the user clicks on the portfolio nav link it should scroll to that part of the page). You can use `href` properties on the nav links to do so.

- Please try to stick as much as possible to the Figma design, which includes:
   - [ ] Implementing the close button shown in Figma and implementing its functionality.
   - [ ] Adding the blur effect to the backdrop of the mobile menu container so that it matches the Figma design.
   - [ ] Removing the text-decoration on the nav links (underlines), so that it matches the Figma template.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/167428131-1ee44fef-021f-4d09-9766-f1c1ba266b7d.png) | ![image](https://user-images.githubusercontent.com/62290545/167428358-9d98ce5c-7519-4eb8-b819-87e69844fd3d.png) |